bc/sponsorship/selectors.py

```diff
@@ -0,0 +1,5 @@
+from .models import Sponsorship
+
+
+def get_available_sponsorship() -> Sponsorship | None:
+    return Sponsorship.objects.filter(current_amount__gte=3.00).first()
```

Comment:
This has magic happening at a distance by relying on the `Meta.order` property. I think it'd be a lot better to put that explicitly here and to remove it from the model. I tend to see default ordering as an anti-pattern because you might rely on it, but you might not, maybe it's doing something, maybe not...you can never be sure and thus you can never change it.

Reply:
got it! I'll use the order_by() method instead.